### PR TITLE
Be specific about which key in constant name

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
Just another tidy...but better to get it in now, I think.

**Comment by jbaublitz** [comment]
Just realized I should take another look at this. Rerequesting review.

File: src/engine/strat_engine/backstore/crypt.rs [hunk 1]
```diff
@@ -109,7 +109,7 @@ impl CryptInitializer {
                 EncryptionFormat::Luks2,
                 ("aes", "xts-plain64"),
                 None,
-                libcryptsetup_rs::Either::Right(STRATIS_KEY_SIZE),
+                libcryptsetup_rs::Either::Right(STRATIS_MEK_SIZE),
```
Comment (jbaublitz) [inline src/engine/strat_engine/backstore/crypt.rs:112]: This is indeed the MEK.

File: src/engine/strat_engine/backstore/crypt.rs [hunk 2]
```diff
@@ -708,7 +708,7 @@ fn read_key(key_description: &str) -> Result<SafeMemHandle> {
             i => i,
         };
 
-        let mut key_buffer = SafeMemHandle::alloc(STRATIS_KEY_SIZE)?;
+        let mut key_buffer = SafeMemHandle::alloc(STRATIS_MEK_SIZE)?;
```
Comment (jbaublitz) [inline src/engine/strat_engine/backstore/crypt.rs:711]: This is not the MEK, it is the passphrase (what generates the KEK). I'd even hesitate to call it the KEK given that it is the passphrase from which the KEK is generated. The two are not the same.

**Comment by mulkieran** [comment]
@jbaublitz Will  you take this PR over and choose good names for the two constants that will disambiguate as necessary? Use "KEK" and "MEK" wherever possible, but where it doesn't apply, choose the name that you believe is most descriptive. Please close this PR when you're ready and open your own.

**Comment by jbaublitz** [comment]
Absolutely!

**Comment by mulkieran** [comment]
Closing, obsolete.

**Pull request closed without merging**